locales/fr.json

```diff
@@ -314,30 +313,29 @@
   "Apply for OTB on the pages that follow.": "Apply for OTB on the pages that follow.",
-  "Enter the last two mailing addresses before your current address": "Enter the last two mailing addresses before your current address",
-  "You must have used both addresses to file taxes. Choose a different question if you haven’t filed taxes from at least two other addresses.": "You must have used both addresses to file taxes. Choose a different question if you haven’t filed taxes from at least two other addresses.",
-  "Mailing address 1": "Mailing address 1",
-  "Street address": "Street address",
-  "Mailing address 2": "Mailing address 2",
-  "Your last two mailing addresses before your current address": "Your last two mailing addresses before your current address",
-  "Enter details about your bankruptcy assignment": "Enter details about your bankruptcy assignment",
-  "Date of your bankruptcy assignment": "Date of your bankruptcy assignment",
-  "Name of your trustee in bankruptcy": "Name of your trustee in bankruptcy",
-  "Details of your bankruptcy assignment": "Details of your bankruptcy assignment",
-  "Your date of entry or release from prison": "Your date of entry or release from prison",
-  "Enter entry or release date from prison": "Enter entry or release date from prison",
-  "Which date do you want to give?": "Which date do you want to give?",
-  "Entry to prison": "Entry to prison",
-  "Release from prison": "Release from prison",
-  "Give the date": "Give the date",
-  "Your bank or credit union account information": "Your bank or credit union account information",
-  "If you can’t answer these questions, yer done boi.": "If you can’t answer these questions, yer done boi.",
-  "Details of your contribution to a Registered Retirement Savings Plan (RRSP)": "Details of your contribution to a Registered Retirement Savings Plan (RRSP)",
-  "Details of your contribution to a Tax Free Savings Account (TFSA)": "Details of your contribution to a Tax Free Savings Account (TFSA)",
-  "Details of your Canada Child Benefit (CCB)": "Details of your Canada Child Benefit (CCB)",
-  "Date and amount of your most recent payment to CRA": "Date and amount of your most recent payment to CRA",
-  "I can’t answer these questions either": "I can’t answer these questions either",
-  "Answer a question from the list below": "Answer a question from the list below",
+  "Enter the last two mailing addresses before your current address": "Vos deux anciennes adresses postales les plus récentes",
+  "You must have used both addresses to file taxes. Choose a different question if you haven’t filed taxes from at least two other addresses.": "Vous devez avoir utilisé ces deux adresses postales pour produire une déclaration de revenus. Choisissez une autre question si vous n’avez pas produit de déclaration de revenus à ces deux adresses.",
+  "Mailing address 1": "Adresse postale 1",
+  "Street address": "Addresse de la rue",
```

Comment:
This should be "Adresse municipale" in French.

Reply:
Good catch 👍